packages/components/src/tooltip/index.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import * as Ariakit from '@ariakit/react/tooltip';
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import { Children } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { TooltipProps } from './types';
+import Shortcut from '../shortcut';
+import { positionToPlacement } from '../popover/utils';
+
+/**
+ * Time over anchor to wait before showing tooltip
+ *
+ * @type {number}
+ */
+export const TOOLTIP_DELAY = 700;
+
+function Tooltip( props: TooltipProps ) {
+	const { children, delay = TOOLTIP_DELAY, position, shortcut, text } = props;
+
+	const baseId = useInstanceId( Tooltip, 'tooltip' );
+	const describedById = text || shortcut ? baseId : undefined;
+
+	const isOnlyChild = Children.count( children ) === 1;
+	// console error if more than one child element is added
+	if ( ! isOnlyChild ) {
+		if ( 'development' === process.env.NODE_ENV ) {
+			// eslint-disable-next-line no-console
+			console.error(
+				'Tooltip should be called with only a single child element.'
+			);
+		}
+	}
+
+	const DEFAULT_PLACEMENT = 'bottom';
+
+	// Compute tooltip's placement:
+	// - compute it from the legacy `position` prop (if defined)
+	// - otherwise, fallback to the DEFAULT_PLACEMENT.
+	let computedPlacement;
+
+	if ( position !== undefined ) {
+		computedPlacement = positionToPlacement( position );
+	} else {
+		computedPlacement = DEFAULT_PLACEMENT;
+	}
+
+	const tooltipStore = Ariakit.useTooltipStore( {
+		// TODO: can remove the ignore once position has been fully deprecated
+		// Type error due to 'overlay' type from positionToPlacement
+		// @ts-ignore
+		placement: computedPlacement,
```

Comment:
Could we simplify this a bit? I mainly want to avoid the `let`.

```suggestion
		placement:
			position === undefined
				? DEFAULT_PLACEMENT
				: positionToPlacement( position ),
```

Reply:
Great suggestion!